products/dns/api.yaml

```diff
@@ -312,6 +312,15 @@ objects:
                   name: 'ipv4Address'
                   required: true
                   description: 'IPv4 address to forward to.'
+                - !ruby/object:Api::Type::Enum
+                  name: 'forwardingPath'
+                  description: |
+                    Forwarding path for this TargetNameServer. If unset or `default` Cloud DNS will make forwarding
+                    decision based on address ranges, i.e. RFC1918 addresses go to the VPC, Non-RFC1918 addresses go
+                    to the Internet. When set to `private`, Cloud DNS will always send queries through VPC for this target
+                  values:
+                  - :default
```

Comment:
It's uncommon but not unheard of for enums to be in lowercase, is this definitely right?

Reply:
confirmed!

Reply:
Capturing our offline conv here for posterity
Similar to ManagedZone: https://github.com/GoogleCloudPlatform/magic-modules/blob/e0c59c4b665bce742c1cf410243a14f5a9fa1d7b/products/dns/api.yaml#L222

Source api docs for this resource: https://cloud.google.com/dns/docs/reference/v1/policies#resource